### ISSUE TITLE
update dependencies

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#166]
While working on #164 I noticed that a lot of the dependencies used by rosetta have newer versions available (not surprisingly), and thought it would be good to update all dependencies while this is being worked on.

**Comment by jbphet** [comment]
I've done this update, and have tested it on ox-dev, but I'm not going to deploy it to the main server until after the Thanksgiving break since I won't be around to fix things if this causes problems.

I was unable to update the version of `pg`.  We are using "^4.5.7" and the currently available version is 7.4.0, so we are quite a ways behind, but I tried updating and it introduced problems with the interaction with `pg-query`.  `pg-query` hasn't been updated at all, and it looks like it's sort of a dead package, so I think the thing to do would be to replace `pg-query` with something else, something more supported.  I'll log an issue for this, but I'm not going to dig into it right now, since we've got bigger fish to fry in rosetta-land.
